doc/api/vm.md

```diff
@@ -10,36 +10,75 @@ The `vm` module enables compiling and running code within V8 Virtual
 Machine contexts. **The `vm` module is not a security mechanism. Do
 not use it to run untrusted code**.
 
-JavaScript code can be compiled and run immediately or
-compiled, saved, and run later.
+JavaScript code can be compiled and run immediately or compiled, saved, and run
+later.
 
-A common use case is to run the code in a different V8 Context. This means
-invoked code has a different global object than the invoking code.
-
-One can provide the context by [_contextifying_][contextified] an
-object. The invoked code treats any property in the context like a
-global variable. Any changes to global variables caused by the invoked
-code are reflected in the context object.
```

Comment:
It doesn’t seem proper to me to remove documentation regarding the currently stable API, and replace it with an experimental one, no matter how good the latter is.

Reply:
Originally this doc deprecated the old context apis and the new one wasn't experimental. I changed that because @joyeecheung blocked on it, but I really don't want to suggest that people continue using the current api.